<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneManagerDialog</class>
 <widget class="QDialog" name="SceneManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="m_tableWidget">
     <column>
      <property name="text">
       <string>Plant Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Position</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visible?</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="m_toggleVisibilityButton">
     <property name="toolTip">
      <string>Toggle the visibility of the selected item</string>
     </property>
     <property name="text">
      <string>Show/Hide Selected</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="m_deleteButton">
     <property name="statusTip">
      <string>This is undoable</string>
     </property>
     <property name="text">
      <string>Delete Selected</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="m_closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
